db/migrate/20160615145518_increase_max_size_of_allowed_emails.rb

```diff
@@ -0,0 +1,5 @@
+class IncreaseMaxSizeOfAllowedEmails < ActiveRecord::Migration
+  def change
+    change_column :communities, :allowed_emails, :text, limit: 16.megabytes - 1
+  end
```

Comment:
Indeed, it raises "ActiveRecord::IrreversibleMigration" if tried to rollback. Good catch. I've been happily using change so many times that forgot to think this line doesn't include the earlier state.